qemu/tests/virtio_console.py

```diff
@@ -65,7 +65,8 @@ def add_chardev(vm, params):
                                     'unix_socket')
         if backend in ['udp', 'tcp_socket']:
             chardev_param['chardev_host'] = host
-            chardev_param['chardev_port'] = str(free_ports[index])
+            _port = params.get('chardev_port_%s' % chardev, str(free_ports[index]))
+            chardev_param['chardev_port'] = _port
```

Comment:
`chardev_param['chardev_port'] = chardev_param.get('chardev_port', str(free_ports[index]))`

Reply:
Hi this line is already included in line above

Reply:
@michiboo i mean replace with `params` with `chardev_param`.

Reply:
Hi it wont work,